src/odemis/dataio/tiff.py

```diff
@@ -2613,3 +2628,116 @@ def _iterDirectories(tiff_file):
             tiff_file.ReadDirectory()
             dir_index += 1
             yield dir_index
+
+ 
+def _parseINIMetadataAsDict(metadata: str) -> dict:
+    """
+    Parses ini metadata as dictionary
+    :param metadata: metadata string as ini  
+    :return: metadata as dictionary
+    """
+    """
+    INI Metadata Format:
+        [subsystem]
+        k=v
+        ...
+    
+    Dict Metadata Format:
+    {
+    subsystem:
+        k: v,
+        ...
+    }
+    """
+    md = {}
+    try:
+        cfg = configparser.ConfigParser()
+        cfg.read_string(metadata)
+        md = {s:dict(cfg.items(s)) for s in cfg.sections()}
```

Comment:
I've inlined the function, but think it should still be converted to a dict as it makes the access clearer. Otherwise, you need to index the lists, which is confusing to read. 

When converted to dict
``` python
date = fei_md["User"]["date"]

```

Using cfg directly
```python
#  (cfg.items("User") -> [('date', '05/09/2023'), ('time', '04:29:30 PM'), ('user', 'Supervisor'), ('usertext', ''), 
# ('usertextunicode', '')])
date = cfg.items("User")[0][1]

```